services/management/dbaas/xtradb_cluster_service.go

```diff
@@ -86,6 +94,30 @@ func (s XtraDBClusterService) ListXtraDBClusters(ctx context.Context, req *dbaas
 	return &dbaasv1beta1.ListXtraDBClustersResponse{Clusters: clusters}, nil
 }
 
+// ShowXtraDBCluster returns a XtraDB cluster.
+func (s XtraDBClusterService) ShowXtraDBCluster(ctx context.Context, req *dbaasv1beta1.ShowXtraDBClusterRequest) (*dbaasv1beta1.ShowXtraDBClusterResponse, error) {
+	kubernetesCluster, err := models.FindKubernetesClusterByName(s.db.Querier, req.KubernetesClusterName)
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO: implement on dbaas-controller side:
+	// 1. Get pxc host and status
+	//  - Ex.: kubectl get -o=json PerconaXtraDBCluster/<cluster_name>
+	// 2. Get root password:
+	//   - Ex.: kubectl get secret my-cluster-secrets -o json  | jq -r ".data.root" | base64 -d
+	_ = kubernetesCluster
+	resp := dbaasv1beta1.ShowXtraDBClusterResponse{
+		Name:     req.Name,
+		Username: "root",
```

Comment:
have you tested this credentials?
Should they be equal to https://github.com/percona/percona-xtradb-cluster-operator/blob/master/deploy/secrets.yaml?

Reply:
Yes, I have tested https://github.com/percona/pmm-managed/pull/490/files#r505539895
Thay equal, just base64 decoded.